commerce_coordinator/apps/enterprise_learner/enterprise_client.py

```diff
@@ -0,0 +1,45 @@
+"""
+API client for calls to Enterprise Service.
+"""
+import logging
+
+from django.conf import settings
+from requests.exceptions import ConnectionError, HTTPError, Timeout  # pylint: disable=redefined-builtin
+
+from commerce_coordinator.apps.core.clients import BaseEdxOAuthClient
+
+logger = logging.getLogger(__name__)
+
+
+class EnterpriseApiClient(BaseEdxOAuthClient):
+    """
+    API client for calls to the Enterprise Service.
+    """
+    enterprise_api_base_url = f"{settings.ENTERPRISE_URL}/enterprise/api/v1"
```

Comment:
We can replace `ENTERPRISE_URL` with `LMS_URL_ROOT` since we call the LMS endpoint for enterprise user check. 

If we still intend to use `ENTERPRISE_URL` then we need to define its value in edx-internal for the stage/prod environment.